misty.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.rdoc_options = ['--charset=UTF-8']
   spec.extra_rdoc_files = %w[README.md LICENSE.md]
 
-  spec.add_dependency 'json', '~> 2.0'
+  spec.add_dependency 'json', '>= 1.8.3', '< 3'
```

Comment:
It seems better to use Ruby's Pessimistic Operator [1] instead to declare deps [2].

Could you please try first:
`spec.add_dependency 'json', '~> 1.8'`

Or if that's not enough then:
`spec.add_dependency 'json', '~> 1.8', '>= 1.8.3'`


[1] https://robots.thoughtbot.com/rubys-pessimistic-operator
[2] http://guides.rubygems.org/patterns/#declaring-dependencies